### PR TITLE
fix add icon being accidentally tiny

### PR DESCRIPTION
**Pull request by jpople**
### Description Of Changes

Fixes add icon on some buttons mistakenly being wrong size.

Changes icon on:

ScrollableList component used in privacy experience/notice forms:

Before:

Screenshot 2024-06-11 at 11 25 42

After:

Screenshot 2024-06-11 at 11 13 53

Data uses tab in system information form (both empty state and when data uses are present):

Before:

Screenshot 2024-06-11 at 11 26 51

After: 

Screenshot 2024-06-11 at 11 27 12

### Code Changes

* [ ] AddIcon component changed to SmallAddIcon in applicable locations
* [ ] Unused CustomFieldsModal and CustomFieldButton components removed

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [ ] Issue Requirements are Met
* [ ] Update `CHANGELOG.md`

### CODE REVIEW

File: clients/admin-ui/src/features/common/custom-fields/index.ts [hunk 1]
```diff
@@ -2,9 +2,7 @@ export * from "./ChooseFromLibrary";
 export * from "./constants";
 export * from "./CreateCustomFields";
 export * from "./CreateCustomLists";
-export * from "./CustomFieldsButton";
 export * from "./CustomFieldsList";
-export * from "./CustomFieldsModal";
```
Comment (eastandwestwind) [inline clients/admin-ui/src/features/common/custom-fields/index.ts:7]: Were these just dangling unused components?

**Review by eastandwestwind — COMMENTED** [review]
Looks good so far! In addition to my question below, I'm wondering about these additional use cases:

* CreateCustomLists.tsx
* CustomFieldForm.tsx

File: clients/admin-ui/src/features/common/custom-fields/index.ts (hunk 1 above)
Comment (jpople) [reply clients/admin-ui/src/features/common/custom-fields/index.ts:7]: Yep, they were exported here but neither was imported anywhere.  ¯\\\_(ツ)\_/¯

**Review by eastandwestwind — APPROVED** [review]
Thanks for clarifying, looks good to me.

**Pull request merged**